src/httpserver.cpp

```diff
@@ -252,7 +252,7 @@ static std::string RequestMethodString(HTTPRequest::RequestMethod m)
 /** HTTP request callback */
 static void http_request_cb(struct evhttp_request* req, void* arg)
 {
-    std::auto_ptr<HTTPRequest> hreq(new HTTPRequest(req));
+    std::unique_ptr<HTTPRequest> hreq(new HTTPRequest(req));
```

Comment:
We're not yet using C++11.  Use boost unique ptr in the meantime.

Reply:
We can't use it. The checks in configure are >= 1.20, unique_ptr lands in 1.57 i'll update patch.